motor/_version.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Version-related data for motor."""
+
+version_tuple = (3, 1, 2, ".dev0")
```

Comment:
`".dev0"` -> `"dev0"`

otherwise version will be '3.1.2..dev0'

Reply:
Or we could fix get_version_string to avoid that by removing leading `.`'s or something.

Reply:
I updated it to account for "rc" releases.

Reply:
How bout we copy what pymongo does to be consistent: https://github.com/mongodb/mongo-python-driver/blob/1d117c1f39983ec88376816c7e3fd73fb140b863/pymongo/_version.py#L21-L24

Pymongo's version is more robust, eg it handles `version_tuple = (3, 1, ".dev0")`